pkg/nvml/api.go

```diff
@@ -553,6 +553,11 @@ type Device interface {
 	VgpuTypeGetMaxInstances(VgpuTypeId VgpuTypeId) (int, Return)
 }
 
+type EventSet interface {
+	Wait(Timeoutms uint32) (EventData, Return)
```

Comment:
`Timeoutms` -> `timeoutms`? (or just keep the types).

The case of the local variables is one of the biggest issues with enabling the linter in this package.

Reply:
I am doing a pass now to lowercase all of the variable names. It was causing problems with `moq` as well. It's the reason this is still a draft PR until I get these updated.